### PR TITLE
Refactor [v104] Organize tests

### PR DESCRIPTION
**Pull request by thatswinnie**
This pull request is organizing all tests and test plans into a test folder.

### CODE REVIEW

File: Tests/XCUITests/SmokeXCUITests.xctestplan [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    }
+  ],
+  "defaultOptions" : {
+    "codeCoverage" : false,
+    "testRepetitionMode" : "retryOnFailure"
+  },
+  "testTargets" : [
+    {
+      "skippedTests" : [
+        "ActivityStreamTest\/testContextMenuInLandscape()",
```
Comment (isabelrios) [inline Tests/XCUITests/SmokeXCUITests.xctestplan:18]: I don't think this is correct, looks like all tests are skipped 🤔

File: Tests/XCUITests/SmokeXCUITests.xctestplan [hunk 2]
```diff
@@ -0,0 +1,197 @@
+{
```
Comment (isabelrios) [inline Tests/XCUITests/SmokeXCUITests.xctestplan:1]: We will need to move also the other testplans:
Smoketest2, Smoketest3, Smoketest4, each one with the tests that have enabled

File: .swiftlint.yml [hunk 3]
```diff
@@ -100,8 +100,8 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
   - Storage/ThirdParty/SwiftData.swift
   - Sync/Generated/Metrics.swift
   - ThirdParty
-  - UITests/
-  - XCUITests/
```
Comment (isabelrios) [inline .swiftlint.yml:104]: Changing path for now, I will work to fix swiftlint in the XCUITests in a follow up PR if that sounds good

**Review by nbhasin2 — APPROVED** [review]

**Comment by nbhasin2** [comment]
This will cause merge conflicts in my PR but you should be able to merge it now :)

**Pull request merged**